index.html

```diff
@@ -378,6 +259,7 @@ <h2 class="contact_title">Contact me</h2>
       </div>
     </footer>
     </div>
+  </div>
     <script src="/js/index.js"></script>
     <script src="/js/formValidation.js"></script>
```

Comment:
Hello @Daphineatim, you are doing a great job because your portfolio looks very nice 💪🏻! However, I would like to mention that the `index.js` file has not been referenced well therefore your portfolio projects fails to load

![image](https://user-images.githubusercontent.com/4247740/158040011-ed892d94-51c0-4653-b40e-ec7439048d52.png)

- [ ] I kindly suggest that you use either of the formats below

`<script src="./js/index.js"></script>` or `<script src="js/index.js"></script>`

Reply:
issue resolved